ground/src/main/java/com/google/android/ground/persistence/remote/TransferProgress.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.persistence.remote
+
+data class TransferProgress constructor(
+    val state: UploadState,
+    val byteCount: Int = 0,
+    val bytesTransferred: Int = 0
+) {
```

Comment:
I think you can omit "constructor"

So this would just be
```
data class TransferProgress(
    val state: UploadState,
    val byteCount: Int = 0,
    val bytesTransferred: Int = 0
) {
```

I've only needed to include the constructor keyword for data classes when I needed to provide overloads for java callers

Reply:
Thanks! Done